cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs

```diff
@@ -1648,12 +1648,16 @@ impl_runtime_apis! {
 	}
 
 	impl sp_genesis_builder::GenesisBuilder<Block> for Runtime {
-		fn create_default_config() -> Vec<u8> {
-			create_default_config::<RuntimeGenesisConfig>()
+		fn build_state(config: Vec<u8>) -> sp_genesis_builder::Result {
+			build_state::<RuntimeGenesisConfig>(config)
 		}
 
-		fn build_config(config: Vec<u8>) -> sp_genesis_builder::Result {
-			build_config::<RuntimeGenesisConfig>(config)
+		fn get_preset(id: &Option<sp_genesis_builder::PresetId>) -> Option<Vec<u8>> {
+			get_preset::<RuntimeGenesisConfig>(id)
+		}
+
+		fn preset_names() -> Vec<sp_genesis_builder::PresetId> {
+			vec![]
```

Comment:
So the number of available presets is `preset_names().len() + 1` since the Default is modelled as `None`?  
I guess its fine.

Reply:
Yes. 

But the `default` itself is not a preset as the other ones. It is a default value which may not guarantee that _runtime_ will work correctly (it may miss some important settings, e.g. initial authorities).  It is a base upon which the presets are applied to create operational initial state.

Initially there was dedicated function for default, but `None` also should work fine here. (ref: https://github.com/paritytech/polkadot-sdk/issues/62#issuecomment-1775078514)

Reply:
Ah i see. I thought that the default would be valid, okay 😄